src/Sylius/Behat/Context/Setup/ChannelContext.php

```diff
@@ -21,33 +23,82 @@
 class ChannelContext implements Context
 {
     /**
+     * @var SharedStorageInterface
+     */
+    private $sharedStorage;
+
+    /**
      * @var DefaultStoreDataInterface
      */
-    private $defaultFranceChannelFactory;
+    private $defaultChannelFactory;
 
     /**
-     * @var SharedStorageInterface
+     * @var DefaultCountriesFactoryInterface
      */
-    private $sharedStorage;
+    private $defaultCountriesFactory;
 
     /**
-     * @param DefaultStoreDataInterface $defaultFranceChannelFactory
      * @param SharedStorageInterface $sharedStorage
+     * @param DefaultStoreDataInterface $defaultChannelFactory
+     * @param DefaultCountriesFactoryInterface $defaultCountriesFactory
      */
     public function __construct(
-        DefaultStoreDataInterface $defaultFranceChannelFactory,
-        SharedStorageInterface $sharedStorage
+        SharedStorageInterface $sharedStorage,
+        DefaultStoreDataInterface $defaultChannelFactory,
+        DefaultCountriesFactoryInterface $defaultCountriesFactory
     ) {
-        $this->defaultFranceChannelFactory = $defaultFranceChannelFactory;
         $this->sharedStorage = $sharedStorage;
+        $this->defaultChannelFactory = $defaultChannelFactory;
+        $this->defaultCountriesFactory = $defaultCountriesFactory;
     }
 
     /**
-     * @Given the store is operating on a single "France" channel
+     * @Given the store is operating on a single channel
```

Comment:
After conversation with @pjedrzejewski we agrees, that there is need of refactoring all country-zone-channel building system, but it should be done in separate PR ;)